src/main/java/examples/CoreExamples.java

```diff
@@ -83,6 +86,50 @@ String blockingMethod(String str) {
     }
   }
 
+  public void exampleFuture1(HttpServer httpServer, NetServer netServer) {
+    Future<HttpServer> httpServerFuture = Future.future();
+    httpServer.listen(httpServerFuture.completer());
+
+    Future<NetServer> netServerFuture = Future.future();
+    netServer.listen(netServerFuture.completer());
+
+    CompositeFuture.all(httpServerFuture, netServerFuture).setHandler(ar -> {
+      if (ar.succeeded()) {
+        // All server started
+      } else {
+        // At least one server failed
+      }
+    });
+  }
+
+  public void exampleFuture2() {
+    Future<String> future1 = Future.future();
+    Future<String> future2 = Future.future();
+    CompositeFuture.any(future1, future2).setHandler(ar -> {
+      if (ar.succeeded()) {
+        // At least one is succeeded
+      } else {
+        // All failed
+      }
+    });
+  }
+
+  public void exampleFuture3(Vertx vertx, Future<Void> startFuture) {
+
+    FileSystem fs = vertx.fileSystem();
+
+    Future<Void> fut1 = Future.future();
+    Future<Void> fut2 = Future.future();
+
+    fs.createFile("/foo", fut1.completer());
+    fut1.compose(string -> {
```

Comment:
The name gives the impression that it is a String, but it is a Void. Maybe void1?